scripts/scripts/vaccinations/automations/incremental/argentina.py

```diff
@@ -1,30 +1,58 @@
-import datetime
 import pandas as pd
-import pytz
 import vaxutils
+import datetime
+import pytz
 
+date = None
 
-def main():
 
-    url = "https://sisa.msal.gov.ar/datos/descargas/covid-19/files/Covid19VacunasAgrupadas.csv"
-    df = pd.read_csv(url, usecols=["primera_dosis_cantidad", "segunda_dosis_cantidad"])
+def read(source: str) -> pd.DataFrame:
+    return pd.read_csv(source, usecols=["primera_dosis_cantidad", "segunda_dosis_cantidad"])
 
-    people_vaccinated = int(df["primera_dosis_cantidad"].sum())
-    people_fully_vaccinated = int(df["segunda_dosis_cantidad"].sum())
-    total_vaccinations = people_vaccinated + people_fully_vaccinated
 
-    date = str(datetime.datetime.now(pytz.timezone("America/Argentina/Buenos_Aires")).date())
+def add_totals(input: pd.DataFrame) -> pd.DataFrame:
+    return input.assign(
+        people_vaccinated=int(input["primera_dosis_cantidad"].sum()),
+        people_fully_vaccinated=int(input["segunda_dosis_cantidad"].sum()),
+    ).assign(
```

Comment:
This would yield a data frame of size `n`, where `n` is the size of the input, where the columns we add have only one value across rows.
Then later we only take the first item of this data frame.

I suggest:

```suggestion
    return (
        input.sum()
        .rename(
            columns={
                "primera_dosis_cantidad": "people_vaccinated",
                "segunda_dosis_cantidad": "people_fully_vaccinated",
                }
            )
        .pipe(enrich_total_vaccinations)
    )
```

This way, it’s evident we care about the resulting series and not the data frame.